### ISSUE TITLE
@babel/cli can't be quietened

### ISSUE THREAD

**Issue by insin** [babel/babel#8326]
# v7 Regression

**Potential Commit/PR that introduced the regression**

https://github.com/babel/babel/pull/7439

**Describe the regression**

`@babel/cli` can no longer be quietened.

The `--quiet` flag was removed and a "Successfully compiled..." message is logged to the console without an option to suppress it:

https://github.com/babel/babel/blob/62ee1adc8543975f1ff0d41dd2ad6875404713a7/packages/babel-cli/src/babel/dir.js#L126-L130

**Environment**

`@babel/cli v7.0.0-beta.53`

**Comment by loganfsmyth** [comment]
It looks like `--quiet` was removed in #7439 but it probably just wasn't fully thought through, not sure. Having it seems reasonable alongside the separate `--verbose` option.

**Comment by Rohanhacker** [comment]
Hey @loganfsmyth, I'd like to work on this. This will be my first contrib. 😃

**Comment by loganfsmyth** [comment]
@Rohanhacker go for it!

**Comment by devenbansod** [comment]
Hi @loganfsmyth @insin so would the `--quiet` option basically serve to remove the currently insuppressible `console.log`?
Also, which one would take precedence if a user passes in both `--quiet` and `--verbose` options?

Am I missing an issue in instead basing the lone `console.log` output to be conditional on `! cliOptions.verbose` and not be required to add `--quiet` back?

**Comment by loganfsmyth** [comment]
Yes, I'd say it should disable all logging. My preference would be for `--verbose` alongside `--quiet` being an error.

> Am I missing an issue in instead basing the lone console.log output to be conditional on ! cliOptions.verbose and not be required to add --quiet back?

I honestly don't feel strongly about it, you're right that we could absolutely just change it so that `--verbose` is required for the "Successfully compiled" message.

@Rohanhacker Have you looked into this at all at this point?

**Comment by Rohanhacker** [comment]
I haven't thought about the case when  --verbose alongside --quiet is used. I was caught up with something. I'll make a pr tomorrow. 😄

**Comment by Rohanhacker** [comment]
@loganfsmyth I've added the quiet flag, Should I make a pr or change it so that --verbose is required for the "Successfully compiled" message ??

**Comment by thallysrc** [comment]
hey @Rohanhacker  do you still working at this issue? i've been looking any pull request opened by you but i have not found it

**Comment by Rohanhacker** [comment]
@thallysrc I've added quiet flag but not sure if it is required or not

**Comment by colinmetcalf** [comment]
Hey @loganfsmyth and @Rohanhacker ! Has this issue been solved yet? If not may I take a look at it? Thanks

**Comment by nbkhope** [comment]
When I upgraded to Babel 7, it stopped console logging the files that were transpiled. How can I get that logging back again? Now it only says `Successfully compiled X files with Babel.`

**Comment by ianstormtaylor** [comment]
Any update on this for adding the `--quiet` flag back?
